src/parser/parser.py

```diff
@@ -92,13 +95,30 @@ def _merge_localizations(self, language, group, data):
             # duplicate key error. This is a temporary measure to keep patch updates going
             elif group != 'heroes':
                 current_value = self.localizations[language][key]
-                print(
-                    f'Key {key} with value {value} already exists in {language} localization '
-                    + f'data with value {current_value}.'
-                )
+
+                # Only bother with an exception if the values are different
+                if current_value != value:
+                    english_value = self.localizations['english'].get(key)
+                    old_is_english = english_value == current_value
+                    new_is_english = english_value == value
+
+                    # if just the old value is english, replace it with the actually localized one
+                    if old_is_english and not new_is_english:
+                        self.localizations[language][key] = value  # use the new localized string
+                    # If just the new value is english, don't replace it
+                    elif new_is_english and not old_is_english:
+                        pass  # keep the old localized string
+                    else:
+                        # If both values are english or neither are english, raise an exception
+                        logger.trace(
+                            f'Key {key} with value {value} already exists'
+                            + f'in {language} localization '
+                            + f'data with value {current_value}.'
+                            + f' English value: {english_value}'
+                        )
```

Comment:
What's the motivation around this?

Reply:
Previously this area threw a hard exception, this *very* old change made it a bit more accurate to the problem it was trying to detect. Now that its just a soft warning its not really necessary and overly specific, will remove.

Reply:
For example, `Key SpiritStealDuration_label with value Spirit Power Steal Duration already exists in ukrainian localization data with value часу духовного ослаблення`
Clearly one of these hasn't been translated yet and the other has, so the implementation was designed to make it use the translated value instead of the untranslated value, instead of whichever one was detected last.